src/bp/ui-shared/src/Form/utils/form.utils.tsx

```diff
@@ -68,7 +71,7 @@ export const formReducer = (state, action) => {
         ...state,
         [key]: updatedItem
       }
-      onUpdate?.(newState)
+      onUpdate?.(newState, field.key)
```

Comment:
quite non standard to pass the key as second argument of a callback .. 
I would expect `key, value` as a developer

Reply:
I know, but you need `newState` 100% of the time, and you might use `key` in like 10% of cases, so I just added it as a secondary parameter. Basically, when you use the `Form`, the `onUpdate` callback will always return the whole object, but you have no way to know which field was changed in the form. 

I added this because maybe you want to do an additional callback if a specific key was changed